src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Family/FamilyTestsHomeCellViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+struct FamilyTestsHomeCellViewModel {
+
+	// MARK: - Init
+
+	init(
+		_ badgeCount: Int = 0
+	) {
+		self.badgeCount = badgeCount
```

Comment:
I think it would be nice to update this by injecting the `FamilyMemberCoronaTestService` and listening to the publisher and then just showing the unseenNewsCount here, updating the cell again using a publisher. That way the cell would not have to be reloaded for a changing badge count